Settings.js

```diff
@@ -156,14 +163,13 @@ function init_settings(){
 			if (toggle.hasClass("rc-switch-checked")) {
 				toggle.click();
 			}
-			window.TOKEN_SETTINGS[setting.name] = false;
+			if (setting.name == 'restrictPlayerMove') toggle.click();
```

Comment:
I don't think we want to default this to `true` because that changes the current default behavior of allowing players to move any token. So I would remove this line.

You're right that we don't need the line that you removed. I think I added that to ensure that every option had a value in the settings object. Otherwise, it's not guaranteed to be there. That's not actually a big deal so removing that line is fine, but I like the idea that if we have a setting we also have a value for it. Thinking through it now, that logic belongs in some sort of "on load" migration process. It's also unnecessary so no need to do it at all. This is a long winded way of saying it's ok to remove that line 😆

Reply:
Alrighty
If it can relieve any concerns, all the TOKEN_SETTINGS are undefined on startup and in practice it acts just like if they were ```false```... but you probably already know that ahaha